projects/story-decorator/src/types.ts

```diff
@@ -9,12 +9,13 @@ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTA
 OF ANY KIND, either express or implied. See the License for the specific language
 governing permissions and limitations under the License.
 */
-import { Color, Scale, ThemeVariant } from '@spectrum-web-components/theme';
+import { Color, Scale, SystemVariant } from '@spectrum-web-components/theme';
 
 declare global {
     interface Window {
         __swc_hack_knobs__: {
-            defaultThemeVariant: ThemeVariant;
+            defaultThemeVariant: SystemVariant;
+            defaultSystemVariant: SystemVariant;
```

Comment:
Can you confirm what this does and whether the fallback process requires maintaining both values?

Reply:
It is not required. Flattened to use only `defaultSystemVariant`